_migration-assistant/migration-phases/deploy/configuration-options.md

```diff
@@ -60,6 +60,12 @@ The following CDK performs a backfill migrations using RFS:
             "userSecretArn": <SECRET_WITH_USERNAME_AND_PASSWORD_KEYS>
         }
     },
+    "// Note": "The following three options are true by default:",
+    "migrationAssistanceEnabled": true,
+    "migrationConsoleServiceEnabled": true,
+    "otelCollectorEnabled": true,
+
```

Comment:
Let's just remove these altogether.  If they're default, no reason to waste the user's attention